schema.sql

```diff
@@ -1,4 +1,7 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals ( id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY, name TEXT NOT NULL, date_of_birth DATE NOT NULL, escape_attempts INT NOT NULL, neutered BOOLEAN NOT NULL,weight_kg DECIMAL NOT NULL );
+CREATE TABLE animals ( id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY, name TEXT NOT NULL, date_of_birth DATE NOT NULL, escape_attempts INT NOT NULL, neutered BOOLEAN NOT NULL,weight_kg DECIMAL NOT NULL);
+
+ALTER TABLE animals
+ADD COLUMN species text;
```

Comment:
#### Good job making the changes @MosDevx, Just a simple change we need to make to make your project reach the primary requirements. 
- [ ] For the  `species` we need to use a string type. with a string/varchar type (you are able to choose the varchar limit that you want), I suggest you go with something like `ALTER TABLE animals ADD  species varchar(255);`